core/domain/services/graphql/mutators/TicketDelete.php

```diff
@@ -85,4 +85,23 @@ public static function deleteTicketAndRelations($entity)
 
         return $result;
     }
+
+    /**
+     * Trashes a ticket.
+     *
+     * @param EE_Ticket $entity
+     * @return bool | int
+     * @throws ReflectionException
+     * @throws InvalidArgumentException
+     * @throws InvalidInterfaceException
+     * @throws InvalidDataTypeException
+     * @throws EE_Error
+     */
+    public static function trashTicket($entity)
+    {
+        // trash the ticket
+        $result = $entity->delete();
+
+        return $result;
```

Comment:
can just return directly

```suggestion
        // trash the ticket
        return $entity->delete();
```